csharp/src/AddressBook/AddressBook.csproj

```diff
@@ -41,6 +41,20 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'ReleaseSigned|AnyCPU' ">

```

Comment:
Doesn't creating a oneof rule in solution configuration make things more complex than they need to be. I like the current clarity and simplicity of the solution configuration that directly translates into project configuration.
That said, I don't feel strongly about this.